CoroutineTemplate/buildSrc/src/main/java/Configurations.kt

```diff
@@ -0,0 +1,15 @@
+object Flavor {
+    const val PRODUCTION = "production"
+    const val STAGING = "staging"
+    const val DIMENSIONS = "stage"
+}
+
+object BuildType {
+    const val DEBUG = "debug"
+    const val RELEASE = "release"
+}
+
+object Module {
+    const val DATA = ":data"
+    const val DOMAIN = ":domain"
+}
```

Comment:
Should we create these objects? I feel like it uses only one place and can be just a string.
Even the Module at [this line](https://github.com/nimblehq/android-templates/pull/138/files#diff-c24cd0177f073063f92b7261a67d0cf5ab26020289bfe8d940b844b413a54b52R2), we didn't use this object. 🤔

Reply:
@lydiasama 
> Even the Module at this line, we didn't use this object.

I did take a FIXME note to mention why we don't use the constants instead. We will need the solution to resolve it in the future.

Reply:
@hoangnguyen92dn That's why I don't think we need to use it as an object 😄 . Shall we use a simple String instead? The same idea to the `Flavor` and `BuildType`.

Reply:
@lydiasama Let's keep it consistent with current approach on RxJava Template as well 🙏